homepage/views.py

```diff
@@ -77,3 +79,22 @@ def add_review_search(request):
             course_name = ''
         courses = Course.get_courses_ordered_by_name(course_name)
         return render(request, 'homepage/add_review_search.html', {'course_name': course_name, 'courses': courses})
+
+
+def sign_up(request):
+    if request.method == "POST":
+        form = SignUpForm(request.POST)
+
+        if form.is_valid():
+            appUser = AppUser()
```

Comment:
please follow the python convention
```suggestion
            app_user = AppUser()
```

Reply:
changed